jobs/mongodb_migration/src/mongodb_migration/collector.py

```diff
@@ -32,4 +35,8 @@ def get_migrations(self) -> List[Migration]:
                 version="20230126164900",
                 description="add 'priority' field to jobs in queue database",
             ),
+            MigrationUpdateSplitNames(
```

Comment:
we should also migrate the jobs in the queue database, no?

Reply:
btw: there are so many past jobs that we don't need anymore that we could delete them periodically. Manually or with a cron job. Otherwise, the migration jobs take forever, and the helm update timeouts and fails. A good reference for that is https://andrewlock.net/deploying-asp-net-core-applications-to-kubernetes-part-7-running-database-migrations/#helm-chart-hooks

> My initial testing of this approach generally worked well, with one exception. If a database migration took a long time, Helm would timeout waiting for the job to complete, and would fail the install. In reality, the job may or may not succeed in the cluster.
> 
> This was a deal breaker for us. Random timeouts, and the reality that production environments (with the larger quantity of data and higher database loads) were likely to be slower for migrations made us look elsewhere. Ultimately we settled on the init container approach I described in the previous section.

Maybe create two issues for that:
1. periodically clean the queue database deleting the old, finished jobs
2. migrate from jobs to init containers to ensure the migration has time to finish without making Helm upgrade timeout

Reply:
Note that for jobs, we have two fields to update, ideally: type and unicity_id

<img width="628" alt="Capture d’écran 2023-02-14 à 15 11 10" src="https://user-images.githubusercontent.com/1676121/218762868-752e48e2-468e-4c26-8ca6-0461bac8d57e.png">

Reply:
Got it, I will send the related code to migration split-names-streaming in another PR